starter-code/javascript/chronometer.js

```diff
@@ -1,10 +1,39 @@
 class Chronometer {
-  constructor() {}
-  // startClick() {}
-  // getMinutes() {}
-  // getSeconds() {}
-  // twoDigitsNumber() {}
-  // stopClick() {}
-  // resetClick() {}
+  constructor() {
+    this.currentTime = 0 ;
+    this.intervalId;
+  }
+
+  startClick(clbk) {
+    this.intervalId = setInterval( () => {
+      this.currentTime ++;
+      let minutes = (this.twoDigitsNumber(this.getMinutes()));
+      let seconds = (this.twoDigitsNumber (this.getSeconds()));
+      clbk(minutes, seconds);
+      }, 1000 );
+  }
```

Comment:
Jasmine is not passing a callback function to test the startClick function, in order to keep the test valid you can test if the callback function is defined before calling it.
```

  startClick(clbk) {
    this.intervalId = setInterval(() => {
      this.currentTime++;
      let minutes = this.twoDigitsNumber(this.getMinutes());
      let seconds = this.twoDigitsNumber(this.getSeconds());
      if (clbk) {
        clbk(minutes, seconds);
      }
    }, 1000);
  }
```